_pytest/python.py

```diff
@@ -1356,6 +1356,9 @@ def __eq__(self, actual):
             return False
         return all(a == x for a, x in zip(actual, self.expected))
 
+    def __hash__(self):
```

Comment:
@The-Compiler see the doc-strings, sequences are supported to make nice expressions with tuples possible